common/src/main/java/com/codeforcommunity/auth/JWTHandler.java

```diff
@@ -76,7 +76,7 @@ private JWTData getJWTDataFromToken(String token) {
     DecodedJWT decodedJWT = getDecodedJWT(token);
     int userId = decodedJWT.getClaim("userId").asInt();
     PrivilegeLevel privilegeLevel =
-        PrivilegeLevel.from(decodedJWT.getClaim("privilegeLevel").asString());
+        PrivilegeLevel.valueOf(decodedJWT.getClaim("privilegeLevel").asString());
```

Comment:
What's the difference between ```valueOf``` and ```from```?

Reply:
`from` was a custom function that we wrote ourselves, but we don't need that anymore since I eliminated the strings from the enum. `valueOf` is built-in and lets us do the same thing, except now we don't have to worry about the uppercase / lowercase differences when dealing with strings representing enums.